docs/source/examples/managed-jobs.rst

```diff
@@ -452,6 +452,286 @@ To submit the pipeline, the same command :code:`sky jobs launch` is used. The pi
   "sky-managed-2022-10-06-05-17-09-750781_pipeline_eval_8-1".
 
 
+.. _pool:
+
+Using pools
+-----------
+
+SkyPilot supports spawning a **pool** for launching many jobs that share the same environment — for example, batch inference or large-scale data processing.  
+The pool consists of multiple individual **workers**, each of which is a SkyPilot cluster instance with identical configuration and setup.  
+All workers in the pool are provisioned with the same environment, ensuring consistency across jobs and reducing launch overhead.
+Workers in the pool are **reused** across job submissions, avoiding repeated setup and **saving cold start time**. This is ideal for workloads where many jobs need to run with the same software environment and dependencies.
+
+
+.. tip::
+
+  To get started with pools, use the nightly build of SkyPilot: ``pip install -U skypilot-nightly``
+
+Create a pool
+~~~~~~~~~~~~~
+
+Here is a simple example of creating a pool:
+
+.. code-block:: yaml
+  :emphasize-lines: 2-4
+
+  # pool.yaml
+  pool:
+    # Specify the number of workers in the pool.
+    workers: 3
+
+  resources:
+    # Specify the resources for each worker, e.g.
+    accelerators: {H100:1, H200:1}
+
+  file_mounts:
+    /my-data:
+      source: s3://my-dataset/
+      mode: MOUNT
+
+  setup: |
+    # Setup commands for all workers
+    echo "Setup complete!"
+
+Notice that the :code:`pool` section is the only difference from a normal SkyPilot YAML.  
+To specify the number of workers in the pool, use the :code:`workers` field under :code:`pool`.  
+When creating a pool, the :code:`run` section is ignored.
+
+
+To create a pool, use the following command:
+
+.. code-block:: console
+
+  $ sky jobs pool apply --pool gpu-pool pool.yaml
```

Comment:
Here and L526+, can we use the shorthand (-p)? Users are likely to pick up whatever the docs recommend.